### ISSUE TITLE
How to use WebDriver for IE11, for exemple to conect with the usual webDriver I only have to direct the path where the .exe is, but for the webDriver It is not possible, as It does not generates a .exe in the same page.

### ISSUE THREAD

**Issue by jhenyffer** [SeleniumHQ/selenium#2188]
## Meta -
OS:  

Selenium Version:  

Browser:  

Browser Version:

**Comment by jhenyffer** [comment]
for exemple to conect with the usual webDriver I only have to direct the path where the .exe is, but for the IE11 webDriver It is not possible, as It does not generates a .exe in the same page.
e.g:
System.setProperty(webdriver.ie.driver, C:\\Selenium\\seleniumIE\\IEDriverServer64.exe);

How can I do this for IE WebDrive, since I don't know the path of the .exe?
Thanks!

**Comment by maninwindow** [comment]
Create a WebDriver folder in your project, put the .exe file of your WebDriver.
For example: the folder name is webDriver, you may write the code like below:
`string webDriver = Path.Combine(System.Environment.CurrentDirectory, "webDriver");`
You don't need to specify WebDriver.exe, just point at the folder name.
Hope it helps you.

**Comment by jhenyffer** [comment]
Thanks, I will try this solution.
Sorry, I am a begginer, and I am with some douts about the code you used:
**string webDriver = Path.Combine(System.Environment.CurrentDirectory, "webDriver");** 

1° doubt - The **Path** is the location of my webdriver? like...
String Path= "C:\\Users\\LocalAdm\\Desktop\\Workspace\\OrangeHRM\\src\\ie\\WebDriver";
String WebDriver = Path.Combine(System.Environment.CurrentDirectory, "WebDriver"); 
2° doubt - I was not able to use **System.Enviroment** don't know way. I am using eclipse.

**Comment by lukeis** [comment]
This is a question rather than an issue. Please send questions to the [selenium user group](https://groups.google.com/forum/#!forum/selenium-users)

For issues please provide a [concise reproducible test case](http://sscce.org/) and describe what results you are seeing and what results you expect.

See [CONTRIBUTING.md](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#issue-contributions)

**Comment by jhenyffer** [comment]
Ok, Thanks. I am going to ask them.

**Comment by maninwindow** [comment]
@jhenyffer  `Path` is from `System.IO.Path`. I am sorry it should not be `System.Environment`, it should be like below:
`string webDriver = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "webDriver");`
`AppDomain` is from `System.AppDomain`.
Good luck.

**Comment by jhenyffer** [comment]
Thanks Alimjan, I am going to try this solution.

**Comment by jimevans** [comment]
@jhenyffer, @maninwindow is using C#. I believe you're using Java. The path items don't directly translate.

**Comment by jhenyffer** [comment]
Thanks guys, I have succeeded with the following....
I used the normal declaration as below:
System.setProperty("webdriver.ie.driver", "C:\\Selenium\\seleniumIE\\IEDriverServer64.exe");
WebDriver driver = new InternetExplorerDriver();
driver.manage().window().maximize();		
driver.get(url);

Added the driver configuration for 64 bits computer  "HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\Internet Explorer\Main\FeatureControl\FEATURE_BFCACHE" to my computer.

I solved It partially, because with 32 bits It runs perfectly, but just on debug mode and is not able to run normally even if I add some wait comands. And with 64 bits driver It fills the fields too slow.
